csharp-basics/exercises/Arrays/Arrays/TicTacToe/Program.cs

```diff
@@ -30,7 +71,60 @@ private static void DisplayBoard()
             Console.WriteLine("  1  " + board[1, 0] + "|" + board[1, 1] + "|" + board[1, 2]);
             Console.WriteLine("    --+-+--");
             Console.WriteLine("  2  " + board[2, 0] + "|" + board[2, 1] + "|" + board[2, 2]);
-            Console.WriteLine("    --+-+--");
+        }
+
+        private static TicTacToeGame CheckIfWinnerExists()
+        {
+            var game = new TicTacToeGame
+            {
+                IsWinnerDefined = false, 
+                IsGameFinished = false
+            };
+
+            for (int i = 0; i < 3; i++)
+            {
+                if (board[i, 0] == board[i, 1] && board[i, 0] == board[i, 2] && board[i, 0] != ' ')
+                {
+                    game.IsWinnerDefined = true;
+                    game.IsGameFinished = true;
+                    game.Winner = board[i, 0];
+                    break;
+                } 
+                
+                if (board[0, i] == board[1, i] && board[0, i] == board[2, i] && board[0, i] != ' ')
+                {
+                    game.IsWinnerDefined = true;
+                    game.IsGameFinished = true;
+                    game.Winner = board[0, i];
+                    break;
+                }
+            }
+
+            if (!game.IsGameFinished)
+            {
+                if (board[0, 0] == board[1, 1] && board[0, 0] == board[2, 2] && board[0, 0] != ' ')
+                {
+                    game.IsWinnerDefined = true;
+                    game.IsGameFinished = true;
+                    game.Winner = board[0, 0];
+                } 
+                else if (board[0, 2] == board[1, 1] && board[0, 2] == board[2, 0] && board[0, 2] != ' ')
+                {
+                    game.IsWinnerDefined = true;
+                    game.IsGameFinished = true;
```

Comment:
Šis ir tāds pats kā iepriekš. VIenādos vajag apvienot, lai nav vairākas reizes jāraksta viens un tas pats

Reply:
Pieļavu kļūdu, tie IF ir dažādi.